src/extension/app/store/app.js

```diff
@@ -594,13 +616,19 @@ export class AppStore {
    * @param {number} [timeout] The timeout in milliseconds (optional)
    */
   showToast(message, variant = 'info', timeout = 2000) {
```

Comment:
The [minimum timeout](https://opensource.adobe.com/spectrum-web-components/components/toast/api/#attributes%20and%20properties_timeout) is 6000.
```suggestion
  showToast(message, variant = 'info', timeout = 6000) {
```